Sources/ArrayExtensions.swift

```diff
@@ -201,6 +203,16 @@ extension Array where Element: Equatable {
     }
 }
 
+extension Array where Element: Hashable {
+
+    /// EZSE: Removes all occurrences of the given object(s)
+    public mutating func removeAll(_ elements: [Element]) {
+        let elementsSet = Set(elements)
```

Comment:
So lets assume the array : val arr = [1, 1, 2, 2, 3] where the elements are hashable. Now, say elements Set = {1, 2, 3}. arr.removeAll(3) would return [1, 2]. 

But the expected contract was [1, 1, 2, 2].